config/default.yml

```diff
@@ -38,8 +38,32 @@ Style/EmptyElse:
 Style/Documentation:
   Enabled: no
 
+Naming/MemoizedInstanceVariableName:
+  EnforcedStyleForLeadingUnderscores: optional
+
+Metrics/MethodLength:
+  Max: 15
+
+Metrics/BlockLength:
+  Enabled: false
+
 RSpec/NoExpectationExample:
   Enabled: no
 
 RSpec/ExampleLength:
   Enabled: false
+
+RSpec/MultipleMemoizedHelpers:
+  Enabled: false
+
+RSpec/MultipleExpectations:
+  Enabled: false
+
+RSpec/NestedGroups:
+  Max: 4
+
+RSpec/MessageSpies:
+  Enabled: false
```

Comment:
This is currently inconsistent across the projects, some places it's disabled, some places it's enforced to 'receive'. Open to arguments.

Reply:
The argument I heard from @rsferreira11 was that since there is no have_received in elixir, then receive would be better for some consistency between the languages.

I personally also prefer have_received but am open to either or disabling it.

Reply:
I think that from testing perspective `have_received` makes more sense because you: setup -> execute -> assert.

But having `expect().to receive` makes you avoid calling `allow` in the test setup, no?

> no have_received in elixir

I have limited knowledge in Elixir, but what I've seen from Mox, you usually just setup a future expectation, similar to `expect().to receive`